src/gui/properties/speedplotview.cpp

```diff
@@ -33,15 +33,61 @@
 #include "base/global.h"
 #include "base/utils/misc.h"
 
+namespace
+{
+    enum PeriodInSeconds
+    {
+        MIN1_SEC = 60,
+        MIN5_SEC = 5 * 60,
+        MIN30_SEC = 30 * 60,
+        HOUR6_SEC = 6 * 60 * 60
+    };
+
+    const int MIN5_BUF_SIZE = 5 * 60;
```

Comment:
I wonder if `constexpr int` will be more effective in all these cases?

Reply:
I guess it won't change a bit in output assembly, also personally I would reserve `constexpr` for more exotic usages.

Reply:
No, it would not. It is already compile-time calculated value. Constexpr just makes wider the domain of expressions able to be compile time calculated, not increases performance.